src/Model/Behavior/SearchBehavior.php

```diff
@@ -30,6 +31,7 @@ class SearchBehavior extends Behavior
             'searchManager' => 'searchManager',
             'isSearch' => 'isSearch'
         ],
+        'emptyValues' => ['']
```

Comment:
This could have one of these key names instead:

- `'filterValues'`
- `'discardValues'`
- `'ignoreValues'`

Then it could make sense to supply a callable from your app and filter arbitrary values.